sync/twinCards/trello.create_twin.js

```diff
@@ -0,0 +1,257 @@
+// ## Webtask `trello.create_twin`
+//
+// This webtask provides automatic card sync between two Trello boards or lists.
+//
+// ### Requirements
+//
+// - This webtask is intended to be triggered by Trello webhooks.
+//   You need to configure Trello's webhooks to call on the webtask URL
+//   (path is simply `/`).
+// - This webtask needs to be capable of performing Trello API calls. For
+//   this, you will have to add the following secrets. You can get both
+//   [here](https://trello.com/app-key).
+//   - `api_key`
+//   - `api_token`
+//
+// - This webtask needs to know the webhook url to add, the list where the
+//   card will be copied and the member ID used as a trigger. For
+//   this, you will have to add the following metas. (use trello sandbox to
+//   get the IDs)
+//   - `webhook_URL` --> url to update_twin functions
+//   - `target_list_ID` --> list where the twin card will be created
+//   - `ref_member_ID` --> user that will be added to a card to be copied
```

Comment:
> user that will be added to a card to be copied

Not really clear. Maybe "user that will trigger the sync when assigned to the card".